src/components/form-editor/form-editor.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Column, Row, Tabs, Tab } from "carbon-components-react";
+import { useTranslation } from "react-i18next";
+import { showToast } from "@openmrs/esm-framework";
+import SchemaEditorComponent from "./schema-editor/schema-editor.component";
+import styles from "./form-editor.scss";
+import { useParams } from "react-router-dom";
+import { useFormSchema, useFormMetadata } from "../../api/fetchFormDetail";
+import { Form, Schema } from "../../api/types";
+
+const FormEditor: React.FC = () => {
+  type State = {
+    uuid: string;
+  };
+  const { t } = useTranslation();
+  const { uuid } = useParams<State>();
+
+  const GetFormMetaData = (formUUID: string) => {
+    const { formMetaData, formMetaDataError } = useFormMetadata(formUUID);
```

Comment:
Any reason as to why hook `useFormMetadata(formUUID)` is not used at the top level of the `FormEditor` functional component?

Reply:
The reason is in a scenario of `Create new form` this wont be needed.

Reply:
I would advise invoking that from the top. I don't see anything special that prevents that..

Reply:
Sure